php/libraries/NDB_Page.class.inc

```diff
@@ -161,14 +163,18 @@ class NDB_Page implements RequestHandlerInterface
 
     /**
      * Wrapper to create a header
+     * FIXME: Setting the name to null is not a good idea because the name
+     *        is the key to the element in the form array. It's been changed
+     *        to a ' ' string so that the method can be unit tested but ideally
+     *        there should be a name param. -Alexandra Livadas
      *
      * @param string $header The text to put in the header
      *
      * @return void
      */
     function addHeader($header)
     {
-        $this->form->addElement('header', null, $header);
+        $this->form->addElement('header', ' ', $header);
```

Comment:
I simply changed the null to ' ' for now because I didn't want to add a name @param in case this method is called in other places in the backend. I'm not sure how to implement this change without having to change every single place this method is called!

Reply:
Where is it breaking that this needs to be changed from null?

Reply:
It's not breaking anywhere but I just thought it was a code smell, since it's not ideal for the name of an element to be null. The name attribute is often the one used to find the element in other methods, so it should not be null.

Reply:
Headers don't have backend names, they're just statically displayed text.. it doesn't make much sense for them to have names of a space to me, and null is reasonable because they have no name.

Reply:
Ok! I can put that back then. In that case, I'll have to change two of the tests I've written.

Reply:
@AlexandraLivadas ping? Doesn't seem to be fixed yet..

Reply:
@driusan just pushed the changes! Let me know how it looks.